tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml

```diff
@@ -13,7 +13,6 @@
             <StackPanel MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
                 <views:FileExplorerView />
                 <views:VersionControlIntegrationSettingsView Visibility="{x:Bind ViewModel.IsFeatureEnabled}"/>
-                <views:AddRepositoriesView Visibility="{x:Bind ViewModel.IsFeatureEnabled}"/>

```

Comment:
The AddRepositoriesView may need to be present for a while inside Dev Home and cannot be removed. Starting an offline email thread for further discussion. Thanks!